Formula/exploitdb.rb

```diff
@@ -1,12 +1,12 @@
 class Exploitdb < Formula
   desc "Database of public exploits and corresponding vulnerable software"
   homepage "https://www.exploit-db.com/"
-  url "https://github.com/offensive-security/exploitdb.git",
-      tag:      "2022-10-18",
-      revision: "b4e29f1fae9fe4aaced5ae0d18ffce260a8165ff"
-  version "2022-10-18"
+  url "https://gitlab.com/exploit-database/exploitdb.git",
+      tag:      "2022-11-22",
+      revision: "225f9878ca58fc0ef8d954ed16a8997af514662b"
+  version "2022-11-22"
   license "GPL-2.0-or-later"
-  head "https://github.com/offensive-security/exploitdb.git", branch: "master"
+  head "https://gitlab.com/exploit-database/exploitdb.git", branch: "master"
```

Comment:
this should be `main`
```suggestion
  head "https://gitlab.com/exploit-database/exploitdb.git", branch: "main"
```

Reply:
Good catch. Thank you!